modules/system-management/pages/workload-management.adoc

```diff
@@ -1,190 +1,181 @@
-|===
-|Field|Description|Data type
+* Running an installed query.
+* Interpreting ad-hoc queries.
+* Executing heavy built-in queries, commonly used for "Explore Graph" in GraphStudio.
 
-|`isEnabled`|The feature flag to enable or disable the workload queue.|`BOOL`
-|`queues`|The map of the available workload queues.|`OBJECT`
-|===
+=== Configure Workload Queues
+Workload queues help control how queries are executed within the system by defining limits such as the maximum number of concurrent queries and the maximum size of the delay queue.
+Administrators can configure workload queues by creating, modifying, or deleting them as needed.
 
-Objects under `queues` consist of queue ID (key) and properties (value).
+Before making changes, it’s a good practice to retrieve the current configuration using the *GET Workload Queue* command. After reviewing, you can update or apply new configurations using the *PUT Workload Queue* command.
 
-CAUTION: The queue ID must be a string of less than 64 characters including alphanumeric and underscore.
+*GSQL Commands*
 
-Each queue has the following properties:
-[cols="20h,~,~"]
-|===
-|Field|Description|Data type
-
-|`description`|The description of the queue.|`STRING` (< 256 characters)
-|`isDefault` (optional)|The flag to indicate if the queue is the default queue. Must be set to `true` for exactly one queue.|`BOOL`
-|`maxConcurrentQueries`|The maximum number of concurrent queries allowed in the queue.|`UINT` in the range of `(0, 131072)`
-|`maxDelayQueueSize`|The maximum number of queries that can be queued in the delay queue.|`UINT` in the range of `[0, 131072)`
-|===
-
-==== maxConcurrentQueries and maxDelayQueueSize
-`maxConcurrentQueries` and `maxDelayQueueSize` are enforced on per machine level. More specifically, it puts a limit on how many requests ONE `GPE` process can handle.
-
-For example, in a TigerGraph cluster with `4` nodes (there will be `4` `GPE` processes), the total number of qureies allowed for a `WorkloadQueue` is `4*maxConcurrentQueries`.
-Similarly, the total number of queries can be put into the corresponding delay queue is `4*maxDelayQueueSize`.
-
-TigerGraph internally would try to evenly distribute queries evenly among the nodes, hence, the `WorkloadQueue` from each `GPE` would be filled at a similar pace.
+* *Retrieve Current Configuration:*
+[source.wrap,gsql]
+----
+GET WORKLOAD QUEUE
+----
 
-[WARNING]
-====
-The query concurrency is also confined by the number of physical cores that the machine has.
-Therefore, `maxConcurrentQueries` of a `WorkloadQueue` is not recommended to be too large (i.e less than `3 * number of machine's physical cores`).
+* *Update Configuration:*
+[source.wrap,gsql]
+----
+PUT WORKLOAD QUEUE FROM "/path/to/queue.json"
+----
 
-Once the configurations change, GPE must be restarted to take effect.
-====
+*REST Endpoints*
 
-===== Examples
+* *Retrieve Current Configuration:*
+[source.wrap]
+----
+GET /gsqlserver/gsql/workload-manager/configs
```

Comment:
```suggestion
GET /gsql/v1/workload-manager/configs
```
@Tushar-TG-14 If this is 4.1, then all the GSQL endpoints have changed: from /gsqlserver/gsql/ to /gsql/v1/, right?